content/source/docs/cloud/vcs/azure-devops-services.html.md

```diff
@@ -43,15 +43,15 @@ Leave the page open in a browser tab. In the next step you will copy values from
 
 ## Step 2: From your Azure DevOps Services Profile, Create a New Application
 
-1. In a new browser tab, open your [Azure DevOps Services Profile](https://aex.dev.azure.com); log in to your Azure DevOps Services account if necessary.
+1. In a new browser tab, open your [Azure DevOps Services Profile](https://aex.dev.azure.com), and log in to your Azure DevOps Services account if necessary. A page with a list of your organizations appears.
 
     ~> **Important:** The Azure DevOps Services account you use for connecting Terraform Cloud must have Project Collection Administrator access to any projects containing repositories of Terraform configurations, since creating webhooks requires admin permissions. It is not possible to create custom access roles with lower levels of privilege, as Microsoft does not currently allow delegation of this capability. If you're unable to load the link above, you can create a new application for the next step at one of the following links: `https://aex.dev.azure.com/app/register?mkt=en-US` or `https://app.vsaex.visualstudio.com/app/register?mkt=en-US`.
 
-1. Click the "Create new application" link at the bottom of the left column under the "Applications and services" header. The next page is a form asking for your company and application information.
+1. Go into your preferred organization, click your user icon in the top right, and then select "More options" > "User Settings" > "Profile". Your profile page appears.
```

Comment:
It might be good to change "more options" to "..." menu or something. Or perhaps include a screenshot?

Reply:
yeah this is a good idea - I'm going to add a screenshot!